draft-ietf-taps-interface.md

```diff
@@ -531,6 +531,12 @@ Transport Properties can have one of a set of data types:
   selection; see {{selection-props}}. When querying, a Preference is of
   type Boolean, with `true` indicating that the Selection Property
   has been applied.
+- Tuple: A specific grouping of multiple value types.
+  In this this document, it is written as a list in brackets, e.g., ```(Enumeration, Preference)``` 
+  The composition of types and the order depends on the property and is fixed for the property.
+  The actual representation is implementation-dependen.
+- Collection: A group of one or more value of the same type.
+  The actual representation, e.g. as a set or an array, is implementation-dependent.
```

Comment:
Not sure we need this sentence. It basically says "The implementation is implementation-dependent", doesn't it? 
Also, a set is not necessarily ordered, while an array is usually ordered, I would say. So I think removing this sentence would make the entire thing less confusing.

Reply:
I still see value in the sentence – especially stressing that the grouping is by design unordered, but might be implemented with some ordered data structure.

Reply:
I'm not sure why it'd be necessary to stress how it could be implemented in the interface draft.
But I'm fine with merging this PR as it is.